### PR TITLE
updated styleprops

### PR DESCRIPTION
**Pull request by kumaryogesh17**
fixes : #14
corrected style props

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -134,13 +134,13 @@ export interface VideoPlayerProps extends VideoProperties {
...
    */
-  videoStyle?: ViewStyle;
+   videoStyle?: StyleProp<ViewStyle>;
 
   /**
    * Container styles
    *
    */
-  containerStyle?: ViewStyle;
+   containerStyle?: StyleProps<ViewStyle>;
```
Comment (criszz77) [inline src/types.ts:143]: There's a typo: it should be `StyleProp`, not `StyleProps` and it should be imported from `react-native`.

```javascript
import type {StyleProp} from 'react-native';
```

**Review by criszz77 — CHANGES_REQUESTED** [review]

File: src/types.ts (hunk 1 above)
Comment (criszz77) [reply src/types.ts]: @kumaryogesh17 Should be `StyleProp` for  `containerStyle` as well.

Comment (criszz77) [reply src/types.ts]: I suggest you to fork the repo locally and run `yarn install`. That way the IDE will let you know about any errors before pushing. 👌

Comment (criszz77) [reply src/types.ts]: Since you also have some eslint/prettier issues now and you won't be able to see them in the web browser :(

Comment (kumaryogesh17) [reply src/types.ts]: Screenshot from 2022-09-09 21-55-32

Comment (criszz77) [reply src/types.ts]: 🙌  That looks good so far, but now we have some `eslint`/`prettier` issues: 

image

Try enabling `eslint` and `prettier` in your editor. You can search about that on Google.

Comment (kumaryogesh17) [reply src/types.ts]: okay

Comment (criszz77) [reply src/types.ts]: More precisely, to solve the first error you'll have to add a space on line 2 before `StyleProp`.

For the second and third error, you'll have to delete the space BEFORE `videoStyle` and `containerStyle` I believe.

But by enabling `eslint` and `prettier` in your editor, they should be fixed automatically by `CTRL + S`

**Review by criszz77 — APPROVED** [review]

**Pull request merged**